forge-gui-mobile/src/forge/screens/match/views/VPlayerPanel.java

```diff
@@ -544,6 +545,12 @@ public void draw(Graphics g) {
                         g.drawText(String.valueOf(experienceCounters), INFO_FONT, getInfoForeColor().getColor(), textStart, (halfHeight*mod)+2, textWidth, halfHeight, false, Align.left, false);
                         mod+=1;
                     }
+                    if (ticketCounters > 0) {
+                        g.fillRect(Color.DARK_GRAY, 0, (halfHeight*mod)+2, INFO_FONT.getBounds(String.valueOf(ticketCounters)).width+halfHeight+1, INFO_FONT.getBounds(String.valueOf(ticketCounters)).height+1);
+                        g.drawImage(FSkinImage.TICKET, 0, (halfHeight*mod)+2, halfHeight, halfHeight);
+                        g.drawText(String.valueOf(experienceCounters), INFO_FONT, INFO_FORE_COLOR.getColor(), textStart, (halfHeight*mod)+2, textWidth, halfHeight, false, Align.left, false);
```

Comment:
Experiencecounters instead of ticketcounters? Also you need to rebase your local copy so it will build without errors (I updated some gui elements so adventure can be themable without affecting the classic version)

Reply:
Thank you!